src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,12 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    public String getRandomBall() {
```

Comment:
why we are returning String when supposed to return Ball?
![image](https://github.com/mogorovskiy/jv-lottery/assets/73041091/e4b20b58-1073-4559-9af1-627fe75758db)

```suggestion
    public Ball getRandomBall() {
```